package.json

```diff
@@ -5,8 +5,6 @@
   "main": "./build/src/index.js",
   "bin": "./build/src/bin/release-please.js",
   "scripts": {
-    "test:node14": "cross-env ENVIRONMENT=test LC_ALL=en c8 mocha --require build/test/_fetch-polyfill.js --recursive --timeout=5000 build/test",
-    "test:node16": "npm run test:node14",
```

Comment:
question: why were these removed?

Reply:
Since https://github.com/stainless-api/release-please/pull/28 this fork of release-please requires node v18+.